`swift run xcodegen` fails

I'm using XcodeGen as a package dependency like

```swift
// swift-tools-version:5.7
// The swift-tools-version declares the minimum version of Swift required to build this package.

import PackageDescription

let package = Package(
    name: "Tools",
    platforms: [.macOS(.v12)],
    products: [],
    dependencies: [
        .package(url: "https://github.com/yonaskolb/XcodeGen", from: "2.38.0"),
    ],
    targets: []
)
```

and run by a command like

```sh
xcrun --sdk macosx swift run --package-path Tools xcodegen
```

Recently, running this command fails by compile error.

```shell
/path/to/project/.build/checkouts/XcodeGen/Sources/XcodeGenKit/PBXProjGenerator.swift:313:20: error: cannot assign to property: 'packages' is a get-only property
        pbxProject.packages = packageReferences.sorted { $0.key < $1.key }.map { $1 }
        ~~~~~~~~~~~^~~~~~~~
[269/272] Compiling XcodeGenKit InfoPlistGenerator.swift
/path/to/project/.build/checkouts/XcodeGen/Sources/XcodeGenKit/PBXProjGenerator.swift:313:20: error: cannot assign to property: 'packages' is a get-only property
        pbxProject.packages = packageReferences.sorted { $0.key < $1.key }.map { $1 }
        ~~~~~~~~~~~^~~~~~~~
[270/272] Compiling XcodeGenKit PBXProjGenerator.swift
/path/to/project/.build/checkouts/XcodeGen/Sources/XcodeGenKit/PBXProjGenerator.swift:313:20: error: cannot assign to property: 'packages' is a get-only property
        pbxProject.packages = packageReferences.sorted { $0.key < $1.key }.map { $1 }
        ~~~~~~~~~~~^~~~~~~~
[271/272] Compiling XcodeGenKit ProjectGenerator.swift
/path/to/project/.build/checkouts/XcodeGen/Sources/XcodeGenKit/PBXProjGenerator.swift:313:20: error: cannot assign to property: 'packages' is a get-only property
        pbxProject.packages = packageReferences.sorted { $0.key < $1.key }.map { $1 }
        ~~~~~~~~~~~^~~~~~~~
```

It seems to be something changed in `PBXProject` class.

--- a/Package.swift
+++ b/Package.swift
@@ -16,7 +16,7 @@ let package = Package(
         .package(url: "https://github.com/yonaskolb/JSONUtilities.git", from: "4.2.0"),
         .package(url: "https://github.com/kylef/Spectre.git", from: "0.9.2"),
         .package(url: "https://github.com/onevcat/Rainbow.git", from: "4.0.0"),
-        .package(url: "https://github.com/tuist/XcodeProj.git", from: "8.13.0"),
+        .package(url: "https://github.com/tuist/XcodeProj.git", exact: "8.13.0"),
         .package(url: "https://github.com/jakeheis/SwiftCLI.git", from: "6.0.3"),
         .package(url: "https://github.com/mxcl/Version", from: "2.0.0"),
         .package(url: "https://github.com/SwiftDocOrg/GraphViz.git", exact: "0.2.0"),

